src/client/pfs/pfs.proto

```diff
@@ -352,4 +360,19 @@ service BlockAPI {
   rpc ListBlock(ListBlockRequest) returns (BlockInfos) {}
 }
 
+message PutObjectRequest {
+  bytes value = 1;
+  repeated Tag tags = 2;
+}
 
+service ObjectAPI {
+  rpc PutObject(stream PutObjectRequest) returns (Object) {}
+  rpc GetObject(Object) returns (google.protobuf.BytesValue) {}
+  rpc GetTag(Tag) returns (google.protobuf.BytesValue) {}
+  rpc Compact(google.protobuf.Empty) returns (google.protobuf.Empty) {}
+}
```

Comment:
In a discussion offline we talked about changing the API to something like:

```
message Object {
  string name = 1;
}

message PutObjectRequest {
  bytes value = 1;
}

message TagObjectRequest {
  Object from = 1;
  Object to = 2;
}

service ObjectAPI {
  rpc Put(stream PutObjectRequest) returns (Object) {}
  rpc Get(Object) returns (google.protobuf.BytesValue) {}
  rpc Tag(TagObjectRequest) returns (google.protobuf.Empty) {}
  rpc Compact(google.protobuf.Empty) returns (google.protobuf.Empty)
}
```

That way we can tag objects retroactively.